WMIAdventure/frontend/src/js/pages/creative/CardsCreator/atoms/CardDescribePreview/CardDescribePreview.js

```diff
@@ -2,26 +2,35 @@ import React from 'react';
 import H1 from './styled-components/H1';
 import P from './styled-components/P';
 import Section from './styled-components/Section';
-import Button from './styled-components/Button';
+import Pencil from "./styled-components/Pencil";
+import Container from "./styled-components/Container";
 
 class CardDescribePreview extends React.Component {
     render() {
         return (
             <>
-                <Section>
-                    <H1>
-                        {this.props.cardName}
-                    </H1>
-                    <P>
-                        {this.props.cardSubject}
-                    </P>
-                    <P tooltip>
-                        {this.props.cardTooltip}
-                    </P>
-                    <Button onClick={this.props.showDescribeInputsHandler}>
-                        {/*pensil icon*/}
-                    </Button>
-                </Section>
+                <Container onClick={this.props.showDescribeInputsHandler}>
+                    {/* Pencil icon */}
+                    <Pencil>
+                    </Pencil>
+
+                    {/* Main content section */}
+                    <Section>

```

Comment:
Jednak jeśli Container miałby być buttonem, to nie może w nim być Section.
Pokombinowałbym tak, że przykleił do sekcji button z tym nagłówkiem za pomocą position absolute, i dał takie wymiary aby wypełniały np. 80% obszaru tego niebieskiego. Pewnie jeszcze trzeba będzie dać temu odpowiednio wysoki z-index przy takim sposobie.

Reply:
Czemu Section nie może być wewnątrz buttona? To jakaś konwencja? W b203dc8 zmieniłem `Container` z `div` na `button` i wszystko działa.

Reply:
Semantyka html'a. Sprawdziłem dla pewności.
![image](https://user-images.githubusercontent.com/51495505/126660188-e23df223-4c2d-49ea-bfaf-ddfe51366794.png)

Możesz sobie sprawdzać na tym https://validator.w3.org/ czy html jest ok.

Reply:
Oo dzięki, przyda się nubowi xD. Skoro nie można `section` to możnaby zamiast tego użyć `div`?

Reply:
div to po prostu taki pojemnik, który nic nie znaczy, section tam jest potrzebne, bo jest tam nagłówek h1

Reply:
rozumiem

Reply:
a nie, jednak się mylę, section to się robi dla dalszych sekcji, przy h2, h1 nie musi być w section, czyli może być w div, ale możliwe że i tak h1 nie powinien być w button

Reply:
z tego co sprawdzam na tym walidatorze to nawet div nie powinien być w button, bezpośrednim dzieckiem button powinna być chyba jedynie treść, ale i tak da się to zrobić, aby button obejmował cały ten obszar bez robienia go rodzicem

Reply:
dlatego też, jak ten ołówek był buttonem, to ta grafika ołówka nie była w <img> tylko zrobiłem ją jako tło

Reply:
spoko, to popróbuję z tym absolute